pymtl3/stdlib/stream/ifcs.py

```diff
@@ -39,9 +100,58 @@ def construct( s, Type ):
 
     s.trace_len = len(str(Type()))
 
+  def connect( s, other, parent ):
+    # SendRTL -> [ RecvRTL -> SendCL ] -> RecvCL (other)
+    if isinstance( other, CalleeIfcCL ):
+      # Create an adapter and add to parents
+      m = RecvRTL2SendCL( s.MsgType )
+
+      if hasattr( parent, "RecvRTL2SendCL_count" ):
+        count = parent.RecvCL2SendRTL_count
```

Comment:
This line seems to be symmetric to the line above. I don't quite get why `RecvCL2SendRTL_count` is used here instead of `RecvRTL2SendCL_count`...